aries-site/src/pages/components/pageheader.mdx

```diff
@@ -0,0 +1,220 @@
+import { Example } from '../../layouts';
+import {
+  ChildPageHeaderExample,
+  PageHeaderActions,
+  PageHeaderAnatomy,
+  PageHeaderContentRegions,
+  PageHeaderIntroExample,
+  PageHeaderResponsiveActions,
+  PageHeaderSubtitle,
+  TopLevelPageHeaderExample,
+} from '../../examples';
+import { Box, Grid } from 'grommet';
+
+The PageHeader's title communicates the context of the page. The subtitle elaborates secondary information about the page. The user is also able to access navigation to the parent page and any relevant page-level actions.
+
+<Example
+  docs="https://v2.grommet.io/pageheader?theme=hpe"
+  height={{ min: 'small' }}
+  showResponsiveControls={[]}
+  template
+>
+  <PageHeaderIntroExample />
+</Example>
+
+## Anatomy
+
+<Example
+  caption="Diagram illustrating the four main content regions of a PageHeader."
+  plain
+>
+  <PageHeaderAnatomy />
+</Example>
+
+1. **Page Identifier:** This region is intended to provide the user with context on what the page is about.
+
+   - **a.** [Title](#title) (required): A short and descriptive title that uniquely identifies the page.
```

Comment:
Title and Subtitle anchor links not taking me to the appropriate sections. It almost looked like the page jumped to a section but took me back to this same exact spot.

Reply:
Good call, strange i'll have to look into that further.

Reply:
Fixed.